client/src/components/Search.css

```diff
@@ -1,33 +1,61 @@
-.search-section {
+/* .search-section {
   display: flex;
   align-items: center;
   justify-content: center;
   margin-bottom: 20px;
-}
+} */
```

Comment:
You shouldn't commit code that has been commented out. It is better to remove it completely